pkg/skaffold/initializer/kubectl.go

```diff
@@ -46,6 +47,8 @@ type kubectlAnalyzer struct {
 
 func (a *kubectlAnalyzer) analyzeFile(filePath string) error {
 	if IsKubernetesManifest(filePath) && !isSkaffoldConfig(filePath) {
+		// to make skaffold.yaml more portable across OS-es we should generate always / based filePaths
+		filePath = strings.ReplaceAll(filePath, string(os.PathSeparator), "/")
```

Comment:
This is causing the failure on windows since the tests expect the `\` as file path separator.
How will this work when a user on window uses `skaffold deploy` and the generated path has `/` instead of expected os.PathSeparator?

Reply:
like we spoke in person. "/" works on windows. resolving this comment